### ISSUE TITLE
Cannot parse KR IDs if title contains year number

### ISSUE THREAD

**Issue by emillon** [tarides/okra#248]
Hi,
The following header confuses okra 1.0.0:

```
- Maintenance - Secret project (2024) (#1234)
```

This works if KR is named

```
- Maintenance - Secret project 2024 (#1234)
```

Thanks

**Comment by jmid** [comment]
Is this the same as https://github.com/tarides/okra/issues/245 - with a fix in #246 and released in 2.0.1? https://github.com/tarides/okra/releases

**Comment by emillon** [comment]
Ah yes I guess so, I missed the 2.0 release

**Comment by emillon** [comment]
Duplicate of #245

**Comment by gpetiot** [comment]
Yes sorry for the chaotic communication this week. There was a retro-compat issue with 1.0 release that had to be fixed in a 2.0 (major number change because of API break).
